argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -107,3 +107,14 @@ spec:
   source:
     repoURL: https://github.com/redhat-appstudio/infra-deployments.git
     targetRevision: pre-kcp
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: enterprise-contract
```

Comment:
Did you mean to name this `enterprise-contract`?

Reply:
Thank you, fixed.